docs/README.md

```diff
@@ -19,7 +20,7 @@ The PlanB Content Repo is designed to serve as a centralized hub for high-qualit
 
 ## Supported Languages
 
-The PlanB content is available in the following languages:
+Thanks to our contributors, the PlanB content is available in many languages such as but not limited to the following ones:
```

Comment:
@Asi0Flammeus does it still make sense to list currently available languages which are visible directly on planb.network website?

Reply:
Moreover, separate projects links are no more available on planb github (old ones are pointing to DecouvreBitcoin repo)

Reply:
Languages paragraph has been completely reviewed